pkg/virt-controller/services/template_test.go

```diff
@@ -1716,9 +1720,13 @@ var _ = Describe("Template", func() {
 
 				pod, err := svc.RenderLaunchManifest(&vmi)
 				Expect(err).ToNot(HaveOccurred())
-				cpu := resource.MustParse("3")
+				cpu := resource.MustParse(expectedCPULimits)
 				Expect(pod.Spec.Containers[0].Resources.Limits.Cpu().Cmp(cpu)).To(BeZero())
-			})
+			},
+				Entry("no annotation added, should allocate one extra emulator CPU", map[string]string{}, uint32(2), "3"),
```

Comment:
Please consider adding an additional entry without the annotation and with an odd number of CPUs as input.

Reply:
sure. DONE.